### PR TITLE
* added method item:isLoadedFromMap

### PR DESCRIPTION
**Pull request by slawkens**

### CODE REVIEW

File: src/item.h [hunk 1]
```diff
@@ -706,6 +706,7 @@ class Item : virtual public Thing
 
 		virtual void startDecaying();
 
+		bool isLoadedFromMap() const {return loadedFromMap;}
```
Comment (ranisalt) [inline src/item.h]: Break lines to keep code style.

Comment (slawkens) [reply src/item.h]: What do you mean? I don't understand.

**Comment by raymondtfr** [comment]
You should create another pull request for the commit ``* fixed client debug when using !z command without parameter``.

File: src/item.h (hunk 1 above)
Comment (raymondtfr) [reply src/item.h]: He meant this:
```C++
bool is LoadedFromMap() const {
    return loadedFromMap;
}
```

**Comment by slawkens** [comment]
Ok, so I've done those line breaks. I suppose you can now merge this?

**Comment by ranisalt** [comment]
> Do I need to create another repository for my other commit?

Please do

> I don't know how can I request new pull request for my second commit.

Use branches.

**Comment by slawkens** [comment]
Gob jub guys. Finally this has been merged!

Thanks.

**Comment by WibbenZ** [comment]
Once a PR has been merged it's "closed".
But you can still comment about it.
@slawkens

**Pull request merged**